deploy/complete/helm-chart/provision/templates/cluster-service-broker.yaml

```diff
@@ -8,5 +8,5 @@ metadata:
   labels:
 {{ include "provision.labels" . | indent 4 }}
 spec:
-  url: http://{{ .Values.global.osb.name }}.{{ .Release.Namespace }}:{{ .Values.global.osb.port }}
+  url: http://{{ .Values.global.osb.name }}.{{ .Values.global.osb.namespace }}:{{ .Values.global.osb.port }}
```

Comment:
Because we are using ClusterServiceBroker instead of ServiceBroker, the name space is not needed on the url. Is there just for compatibility with the old documentation. This can easily be changed to url: http://{{ .Values.global.osb.name }}:{{ .Values.global.osb.port }} when we install the OSB on the mushop-setup chart

Reply:
New commit removed namespaces service broker

Reply:
I've been testing the steps locally and feel this has introduced a problem. It has been tested both locally and in OKE with the helm OCI-OSB installation below, and will return an error fetching catalog in all cases _except_ where the namespace is included or added manually.

```shell
helm install mushop-osb https://github.com/oracle/oci-service-broker/releases/download/v1.3.3/oci-service-broker-1.3.3.tgz \
  --set ociCredentials.secretName=oci-credentials \
  --set storage.etcd.useEmbedded=true
  --set tls.enabled=false
```

> The change to `mushop-osb` also needs to be documented, or we need to revert the change for `global.osb.name`

Then install provisioning chart

```
helm install provision provision ...
```

Then check the csb...

```text
k get clusterservicebrokers
mushop-osb   http://mushop-osb:8080   ErrorFetchingCatalog   25s
```

```text
k describe clusterservicebroker/mushop-osb
// ...
Events:
  Type     Reason                Age                 From                                Message
  ----     ------                ----                ----                                -------
  Warning  ErrorFetchingCatalog  10m (x13 over 23m)  service-catalog-controller-manager  Error getting broker catalog: Get http://mushop-osb:8080/v2/catalog: dial tcp: lookup mushop-osb on 10.96.0.10:53: no such host
```

So edit url to specify namespace of `oci-service-broker` chart installation...

```text
k get clusterservicebrokers
mushop-osb   http://mushop-osb.mushop:8080   Ready  15s
```

Reply:
The "Cluster" Service Broker should not be installed to the namespace mushop. The idea to use "mushop-utilities" are for any supporting services, including service-catalog (Already installed in a separate namespace together with prometheus, Grafana, etc... Service Broker should be there, as actually is not a Microservices, but instead is a supporting service.

Reply:
The osb should only be namespace "locked" if you use ServiceBroker instead of ClusterServiceBroker

Reply:
but `ClusterServiceBroker` is never assigned a namespace. Even when using the `provision` chart. The flow here is that `ClusterServiceBroker` has a `url` setting which must resolve to the service started by the `oci-service-broker` chart (which requires namespace).

Reply:
That's why service Catalog and OSb should be in the same namespace, to avoid this limitation of the OSB.

Reply:
PR #84 includes the documentation changes.

Reply:
So, essentially you're saying that the solution here is that `svc-cat/catalog` and `oci-service-broker` need to be installed in the same namespace